yellowfinLib.py

```diff
@@ -719,9 +736,18 @@ def loadPPKdata(fldrlistPPK):
             #     Tpos = pd.read_csv(fn, sep="\s{2,}", header=10, names=colNames, engine="python")
             # except ValueError:
             #     Tpos = pd.read_csv(fn, sep="\s{2,}", header=12, names=colNames, engine="python")
+
+            # Determines how many rows to skip in the POS file
+            with open(fn, 'r') as f:
+                skip_rows = 0
+                for line in f:
+                    if line.lstrip().startswith('202'):  # crude date check
```

Comment:
This makes an assumption of data collected in 2020's? Is there a better line identifier/starter ?